x-pack/plugins/apm/server/routes/service_map/get_service_stats.ts

```diff
@@ -49,6 +50,7 @@ export async function getServiceStats({
             ...environmentQuery(environment),
             ...termsQuery(SERVICE_NAME, serviceName),
             ...kqlQuery(serviceGroupKuery),
+            ...kqlQuery(kuery),
```

Comment:
nit: perhaps more specific naming (consistent with `serviceGroupKuery`)
```suggestion
            ...kqlQuery(searchBarKuery),
```